README.md

```diff
@@ -1 +1,55 @@
 # MediaMonks - Frontend Coding Standards
+
+## Definition of Done Checklist
+
+#### Before Starting feature
+- [x] Read the ticket. If no ticket is present please ask the PM for one.
+- [x] Make sure the ticket is clear and actionable. If not, reassign the ticket to your project manager until the ticket is 100% clear.
+- [x] Create a feature branch (`feature/ticket-number-feature-name`)
```

Comment:
We can add to the example `feature/ticket-number-feature-name` that for Jira projects, to use the default branch name when created from a ticket